### DISCUSSION TITLE
Getting error : Unknown authentication plugin `sha256_password'. with MySQL 5.7.33

### DISCUSSION THREAD

**Discussion by ansarizeeshan** [prisma/prisma#14225] [Q&A]
### Bug description

I am not sure, why this bug is moved to diuscussion.

[https://github.com/prisma/prisma/discussions/14181](url)

I am getting below error message:
````
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "db" at "host:port"

Error: Migration engine error:
Error querying the database: Unknown authentication plugin `sha256_password'.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
````
MySQL config is having below flag as:
<html>
<body>

default_authentication_plugin | sha256_password
-- | --

</body>
</html>

### How to reproduce

Using MySQL version 5.7.33 with below config/flag enabled:

<html>
<body>

default_authentication_plugin | sha256_password
-- | --

</body>
</html>

### Expected behavior

It should run and connect to database successfully .

### Prisma information

````
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model Account {
}
````

### Environment & setup

- OS: Windows
- Database: MySQL,
- Node.js version: v14.18.2

### Prisma Version

```
"prisma": "^4.0.0",
"@next-auth/prisma-adapter": "^1.0.3",
"@prisma/client": "^4.0.0",
```

**Comment by janpio** [comment]
Because we do not think it is a bug.

Please see and finish the discussion at https://github.com/prisma/prisma/discussions/14181

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
